src/devices/cpu/ht1130/ht1130.cpp

```diff
@@ -170,7 +170,8 @@ void ht1130_device::internal_data_map(address_map &map)
 {
 	map(0x00, 0x9f).ram().w(FUNC(ht1130_device::tempram_w)).share("tempram");
 
-	// the ht1130 is meant to have displayram from 0xe0, so this could be a newer chip?
+	// the HT1130 is meant to have displayram from 0xe0, so this could be a newer chip?
+	// HT1190 has data 00-9f and display b0-ff, possibly a closer match?
```

Comment:
Sean when he read that Russian article today: "The pad locations don’t match those shown in the HT1130 datasheet.  They do match HT1190."

Reply:
yeah, it seems logical as it doesn't seem to be using a0-af either, let me adjust things for that

Reply:
right, I've set it to HT1190.  the CPU core folder is still called 1130, but there's a derived class for the 1190